src/api/models/ActionModel.js

```diff
@@ -0,0 +1,27 @@
+const mongo = require("mongoose");
+
+const ActionSchema = new mongo.Schema({
+  name: { type: String, required: true },
+  description: { type: String },
+  type: { type: [mongo.Schema.Types.ObjectId] },
+  location: {
+    type: {
+      type: String,
+      enum: ["Point"],
+      required: true
+    },
+    coordinates: {
+      type: [Number],
+      required: true
+    }
+  },
+  date: { type: Date },
+  photo: { type: String },
+  organizer: {
+    _id: { type: mongo.Schema.Types.ObjectId },
```

Comment:
I think a more fitting name is organizerId, to avoid confusion with MongoDB's _id